core/src/tasks/system/status/status.ts

```diff
@@ -81,6 +81,7 @@ export class StatusTask extends CLSTask {
     if (!toStop) return false;
 
     // do not await so the promise can end so the server can shut down
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
It really does not like returning a promise with an async call. Looks like this is intentional but could be dangerous otherwise. Doc explains why:

[no-async-promise-executor](https://eslint.org/docs/rules/no-async-promise-executor#no-async-promise-executor)

Reply:
The proimise isn't returned. Does:
```
await api.process.stop();
process.nextTick(() => process.exit(0));
```
Even need to be wrapped?

I feel like maybe the entire block could be `api.process.stop().then(() => process.nextTick(() => process.exit(0)));`
@evantahler Thoughts?